docs/new_predicate_guidelines.md

```diff
@@ -49,9 +49,10 @@ specification for you.
 
 ## Vetting process
 
-Our vetting process is simple.
+Our vetting process is simple, and is based on the process specified in [ITE-9].
 
-1.  Open a PR following [ITE-9] formatting guidelines.
+1.  Open a PR following the formatting guidelines specified in the
```

Comment:
also linking to the existing predicates that people can look at as examples would be helpful for first time users.